Service/Request/RequestFactory.php

```diff
@@ -210,4 +211,17 @@ private function trimCurlyBrackets($str)
     {
         return trim($str, '{}');
     }
+
+    /**
+     * @param array $queryParamsString
+     *
+     * @return array
+     */
+    private function normalize_queryParams($queryParamsArray)
```

Comment:
We should use "camel case" format for function names

Reply:
Fixed.